### ISSUE TITLE
Shared memory error when starting

### ISSUE THREAD

**Issue by brechtcs** [beakerbrowser/beaker#1701]
I'm trying to run Beaker from source using `npm start`, but I get the following error:

```
info:     Logger started
info:     Welcome to Beaker {"category":"browser","subcategory":"init"}
info:     Initializing databases {"category":"browser","subcategory":"init"}
info:     Starting hyperdrive {"category":"browser","subcategory":"init"}
[2605:0601/151643.817376:FATAL:platform_shared_memory_region_posix.cc(254)] This is frequently caused by incorrect permissions on /dev/shm.  Try 'sudo chmod 1777 /dev/shm' to fix.
info:     Connected to an external daemon. {"category":"hyper","subcategory":"daemon"}
info:     Initialized dat daemon {"category":"hyper","subcategory":"drives"}
info:     Initializing hyperdrive filesystem {"category":"browser","subcategory":"init"}
info:     Loading root drive {"category":"hyper","subcategory":"filesystem","url":"hyper://1968..a4/"}
info:     Initializing browser {"category":"browser","subcategory":"init"}
info:     Registering protocols {"category":"browser","subcategory":"init"}
info:     Running setup flow {"category":"browser","subcategory":"init"}
[2620:0601/151644.276132:FATAL:platform_shared_memory_region_posix.cc(254)] This is frequently caused by incorrect permissions on /dev/shm.  Try 'sudo chmod 1777 /dev/shm' to fix.
```
I don't know if it's relevant, but to work around an earlier problem I did add a `--no-sandbox` flag to the Electron child process in `scripts/tasks/start-cli.js`.

 - OS: Debian Stretch
 - Version: git master

**Comment by pfrazee** [comment]
Huh. I wonder why you have to do that on linux and I don't. I'm using Ubuntu and running from source doesn't have any problems for me

**Comment by brechtcs** [comment]
Okay, so I tried to remove the `--no-sandbox` flag today, so I could share the error in this thread. But then it just worked.

I haven't tried to reproduce it yet, but I thinks I just forgot to run `npm run rebuild`. 

What probably happened is I got an Electron error mentioning sandbox issues that would have been fixed by a rebuild. But instead of doing that I googled the error and found a suggestion to use `--no-sandbox`. Adding that flag gave another error which I did solve correctly by using `npm run rebuild`. After that though the shared memory issue caused by `--no-sandbox` surfaced. Removing the flag again today was the fix for that final problem.

Basically, I thought I didn't need to RTFM, because I'd built Beaker dozens of times before. Turns out I was wrong. I just hope this post-mortem will be of help when someone else gets stuck in the same rabbit hole.
